### DISCUSSION TITLE
Copilot completions blocked by other extensions

### DISCUSSION THREAD

**Discussion by noxabellus** [community/community#128683] [Copilot Conversations]
### Select Topic Area

Bug

### Body

https://github.com/haskell/vscode-haskell/issues/1104

While installing the previous version of vscode-haskell fixes the problem, that extension hasn't been updated in a month but this problem cropped up for me in the past few days. I think the most recent updates to Copilot are possibly the actual cause of the incompatibility here.

**Comment by samannaz001** [comment]
I noticed the same issue recently. Rolling back to the previous version of vs code-Haskell helped, but I agree, it seems like the recent Copilot updates might be causing this problem.
